README.md

```diff
@@ -2,35 +2,31 @@
 
 ## Microprofile based Microservice Apps Integration with MySQL Database Server
 
-This repository contains the **MicroProfile** implementation of the **Orders Service** which is a part of the 'IBM Cloud Native Reference Architecture' suite, available at https://github.com/ibm-cloud-architecture/refarch-cloudnative-kubernetes
+This repository contains the **MicroProfile** implementation of the **Orders Service** which is a part of the 
+'IBM Cloud Native Reference Architecture' suite, available at https://github.com/ibm-cloud-architecture/refarch-cloudnative-kubernetes
```

Comment:
What is the change here?

Reply:
Divided to fit 80 character ruler as a preference.
Doesn't affect outward appearance on the page.